pegasus/sites.v3/code.org/public/learn/index.haml

```diff
@@ -27,25 +44,16 @@ social:
     - else
       = view :header, variant: "mobile"
 
-  .banner
-    .container_responsive{style: "color: white"}
-      .col-50{style: "text-align: center"}
-        %img{src: "/images/learn/hour-of-code-logo-2016.png", style: "margin-top: 30px; margin-bottom: 30px; max-width: 100%; padding-left: 5px; padding-right: 5px;"}
-        %br/
-        The Hour of Code is a global movement reaching tens of millions of students in 180+ countries. Anyone, anywhere can organize an Hour of Code event. One-hour tutorials are available in over 45 languages for all ages.
-        %br/
-        %br/
+  .banner.container_responsive
+    .col-50
+      .bannerHeading= hoc_s(:learn_banner_heading)
+      = hoc_s(:learn_banner_blurb)
+      .bannerButtons
         %a{href: "https://hourofcode.com/#join", target: "_blank"}
-          %button
-            Host an Hour
-        &nbsp;
-        &nbsp;
-        %a{href: "https://hourofcode.com/how-to", target: "_blank"}
-          %button
-            Teacher Guide
-        %br/
-        %br/
-      #studentimage.col-50.tablet-feature{style: "height: 278px"}
+          %button= hoc_s(:learn_banner_host)
+        %a{href: "https://hourofcode.com/how-to", target: "_blank", style: "padding-left: 20px"}
```

Comment:
Do we have a CDO.hoc_url() function? Not needed here but may be handy for these pages in the future since we do a lot of jumping around.